### DISCUSSION TITLE
Is Iris' fork of Sodium getting merged with upstream Sodium?

### DISCUSSION THREAD

**Discussion by BeansBeefBroccoli** [IrisShaders/Iris#985] [Q&A] [answered]
The Iris fork of Sodium just got archived, so I was wondering if it's just been abandoned or if it's gonna get merged.

**Comment by AlsoSylv** [comment, marked as answer]
Iris no longer requires a fork of sodium, and will work with sodium 0.3.2 official next release

**Reply by BeansBeefBroccoli** [reply]
Ok ty!
